package.json

```diff
@@ -46,6 +46,7 @@
     "semver": "^7.4.0",
     "ts-jest": "^29.1.1",
     "tsup": "^7.1.0",
-    "typescript": "^5.1.6"
+    "typescript": "^5.1.6",
+    "viem": "^1.16.6"
```

Comment:
Added `viem` mostly for the `parseEther` and other utilities, we will probably use them more often in unit tests